src/components/ChevronKnapp.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Chevron from 'nav-frontend-chevron';
+import styled from 'styled-components';
+
+const StyledChevronKnapp = styled.button`
```

Comment:
Om vi skal bruke prefix/postfix for disse, så synes jeg det er lettere å lese med postfix enn prefix, ettersom det viktigste kommer først.
Tanker @JMLindseth @torchr89 @juneih ?

Reply:
Forsåvidt enig i det.

Reply:
Hvis det viktigste skal komme først, er det ikke prefix som er lettere å lese? Eller mener du at styled er det viktigste? @janzh  #forvirra

Reply:
Jeg er usikker på disse; er det nødvendig at de skal hete `styled`? Det er jo ikke så veldig viktig å vite at en komponent er stylet? Men jeg synes det også blir vanskelig å gi de fornuftige navn, fordi hovedkomponenten får det beste navnet. 🤷‍♂